tutorial/chapter1.md

```diff
@@ -0,0 +1,96 @@
+<p align="right">
+  <img width="25%" height="25%"src="../images/klepsydra_logo.jpg">
+</p>
+
+# Introduction
+Klepsydra vision API is split into main groups also (as well as Klepsydra API):
+
+* **Application API**
+* **Composition API**
+
+# mem example
+
+In this example the resize of five images is done. 
```

Comment:
This example shows the usage of the DataMultiplexer (from kpsr-core) when using opencv `Mat` images using ImageDataFactory and ImageMult.... In this sample example, we process images by resizing them to desired size and publish them.

Reply:
Done